### PR TITLE
NodeJS week 2 homework

### PR DESCRIPTION
**Pull request by cecastosic**

### CODE REVIEW

File: nodejs/week2/meal-sharing/src/backend/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    console.log(`${new Date().toLocaleString()} request received for path: ${req.path}`);
+    next();
+});
+
+app.use('/meals', mealsRouter);
+app.use('/api/meals', mealsApiRouter);
+app.use('/reservations', reservationsRouter);
+app.use('/reviews', reviewsRouter);
+
+app.get('/', (req, res) => res.send('Error no page found'));
```
Comment (arcevit) [inline nodejs/week2/meal-sharing/src/backend/index.js]: here it is a mains page
app.get('/', (req, res) => res.send('Error no page found'));
it is better to use the main page or some relevant word

File: nodejs/week2/meal-sharing/src/backend/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const express = require("express");
```
Comment (arcevit) [inline nodejs/week2/meal-sharing/src/backend/index.js]: Hi Svetlana
I have reviewed your code for 2 nd week of node.js homework
it is quite nice and clean code with naming conventions 
You have implemented extra feature by appending reviews to meal.js
:+1:

Comment (cecastosic) [reply nodejs/week2/meal-sharing/src/backend/index.js]: Thank you for your feedback!

File: nodejs/week2/meal-sharing/src/backend/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+const reviewsRouter = require('./routes/reviews.js');
+
+// make a middleware the logs the the date and the request's method, each time the server gets a request
+app.use((req,res, next) => {
+    console.log(`${new Date().toLocaleString()} request received for path: ${req.path}`);
+    next();
+});
+
+app.use('/meals', mealsRouter);
+app.use('/api/meals', mealsApiRouter);
```
[the remark this answers is not in the document]
Comment (cecastosic) [reply nodejs/week2/meal-sharing/src/backend/index.js]: It wasn't clear from the homework task, should the route be /api/meals or /meals, therefore I made both. But I removed /meals now, and left /api/meals.

**Comment by cecastosic** [comment]
Thanks @moonflare  for the detailed feedback! I think I resolved everything. 👍 
Thanks @arcevit for the feedback also.

**Pull request merged**